components/date_picker/DatePicker.js

```diff
@@ -70,40 +71,45 @@ const factory = (Input, DatePickerDialog) => {
       }
     }
 
+    toggleActive = (active) => {
+      if (this.props.onActiveChange) this.props.onActiveChange(active);
+      this.setState({ active });
```

Comment:
Since calling `onActiveChange` happens right after the actual state is changed, maybe it makes sense to use the callback parameter of `setState` to perform this action:

```jsx
this.setState({ active }, () => {
  if (this.props.onActiveChange) {
    this.props.onActiveChange(active);
  }
})
```